### ISSUE TITLE
Could not display aliases

### ISSUE THREAD

**Issue by vvrs** [awulkiew/graphical-debugging-vscode#9]
I have added `boost.json` to my workspace with the following content, 

```
{
    "name": "graphicaldebugging",
    "language": "cpp",
    "aliases": [
        {
            "name": "bg_types::Point",
            "type": "boost::geometry::model::point<double,2,boost::geometry::cs::cartesian>"
        },
        {
            "name": "bg_types::Polygon",
            "type": "boost::geometry::model::polygon<boost::geometry::model::point<double,2,boost::geometry::cs::cartesian>,true,true,std::vector,std::vector,std::allocator,std::allocator>"
        }
    ]
}
```
But that doesn't seem to be working. Does `*.json` file need to have any specific name?

**Comment by awulkiew** [comment]
I have't tried to reproduce it yet but alias name should be exactly the same as it is reported by the debugger. Are namespaces included in type definition e.g. in Locals window?

**Comment by awulkiew** [comment]
No, any filename should be ok as long as it is *.json file which contains `name` and `language`.

I tried to reproduce it with this program:
```
#include <boost/geometry.hpp>

namespace bg = boost::geometry;

namespace bg_types {
    using Point = bg::model::point<double, 2, bg::cs::cartesian>;
    using Polygon = bg::model::polygon<Point>;
}

int main()
{
    bg_types::Point pt{1, 2};
    bg_types::Polygon po{{{0, 0}, {0, 1}, {1, 1}, {1, 0}}};

    return 0;
}
```

and your `boost.json` but it works for me:

![image](https://user-images.githubusercontent.com/1226951/217690543-67d1bb7e-f2e7-47b9-be4b-5dc41c0da50d.png)
![image](https://user-images.githubusercontent.com/1226951/217690797-7029607c-b4e6-4229-98a3-896cd6648fb6.png)

**Comment by awulkiew** [comment]
@vvrs what compiler and debugger are you using?

**Comment by awulkiew** [comment]
I'll close it for now. If you want to continue the discussion just reopen it.

**Comment by vvrs** [comment]
It is taking too long, but it is working. Thanks for the response.

**Comment by awulkiew** [comment]
@vvrs Automatic alias unrolling for GDB and LLDB is released with version 0.8 along with other improvements for C++. You shouldn't be forced to define aliases manually anymore. Could you give it a try?
